src/Symfony/Component/Validator/Resources/translations/validators.gl.xlf

```diff
@@ -192,7 +192,7 @@
             </trans-unit>
             <trans-unit id="51">
                 <source>No temporary folder was configured in php.ini.</source>
-                <target>Ningunha carpeta temporal foi configurada en php.ini.</target>
+                <target>Ningunha carpeta temporal foi configurada en php.ini, ou a carpeta non existe.</target>
```

Comment:
Why does this one needs to be modified?

Reply:
That's because the English version has it
```
<target>No temporary folder was configured in php.ini, or the configured folder does not exist.</target>
```
That `or the configured folder does not exist` means `ou a carpeta non existe`